### DISCUSSION TITLE
ZBMini external switch/trigger mode

### DISCUSSION THREAD

**Discussion by FabioEight** [Koenkk/zigbee2mqtt#8675] [Devices]
Hello,

I've several ZBMinis installed in my home, connected to lights via pulse mode.
Using the Mini R2 (wifi), I'm able to change the operational mode of the device, setting it to "Pulse Mode".
I checked the Z2M dashboard, but I don't see any configuration tab for my ZBMini (zigbee). Is it possible to configure the external switch mode via z2m? Or in this case I need to use the proprietary bridge (zbridge) with the companion app (eWelink) to configure the external switch modes?
I'm currently using a Conbee 2 (II).

Thanks!

**Comment by francisp2** [comment]
What is "Pulse mode" ? if you want to use the zbmini with a momentary switch, that won't work (hardware does not support it, mini or mini r2 do) The zbmini can only be used with rocker switches. See the product specifications on the Sonoff site.

**Comment by nomis** [comment]
The ZBMINI (date code 20200927) already supports momentary switches. Just toggle the rocker switch twice quickly and you'll find it only toggles the light state once. I don't know why their website and user manual don't mention this.
